app/src/main/res/values/dimens.xml

```diff
@@ -7,4 +7,11 @@
     <dimen name="sp_30">30sp</dimen>
     <dimen name="dp_125">125dp</dimen>
     <dimen name="dp_150">150dp</dimen>
+    <dimen name="dp_100">100dp</dimen>
+    <dimen name="sp_15">15sp</dimen>
+    <dimen name="sp_20">20sp</dimen>
+    <dimen name="dp_8">8dp</dimen>
+    <dimen name="dp_350">350dp</dimen>
+    <dimen name="dp_30">30dp</dimen>
+    <dimen name="dp_70">70dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_70` appears to be unused
 Unused resources make applications larger and slow down builds.